chapter06/src/main/java/com/seaofnodes/simple/node/NotNode.java

```diff
@@ -0,0 +1,28 @@
+package com.seaofnodes.simple.node;
+
+import com.seaofnodes.simple.type.Type;
+import com.seaofnodes.simple.type.TypeInteger;
+
+public class NotNode extends Node {
```

Comment:
Are we keeping this or as @XmiliaH  suggested, replace it with existing node?

Reply:
Parser.TODO shows up.
Could be moved to e.g. Node.TODO?